prrte/cycle/run.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Final return value
+FINAL_RTN=0
+
+# Number of nodes - for accounting/verification purposes
+NUM_NODES=${CI_NUM_NODES:-1}
+
+NUM_ITERS=1000
+NUM_ITERS=10
+
+# ---------------------------------------
+# Start the DVM
+# ---------------------------------------
+if [ "x" = "x$CI_HOSTFILE" ] ; then
+    prte &
+else
+    prte --hostfile $CI_HOSTFILE &
+fi
+
+# Wait for DVM to start
+sleep 5
+
+# ---------------------------------------
+# Run the test
+# ---------------------------------------
+rm output.txt ; touch output.txt
+for n in $(seq 1 $NUM_ITERS) ; do
+    echo "---------------------"
+	echo "Execution $n"
+    echo "---------------------"
+	prun -n 1 hostname 2>&1 | tee -a output.txt
```

Comment:
This test is also failing most of the time. 10 iterations is usually enough to get it to wedge with an error like:
```
[d81fb6f346bb:00253] PRRTE ERROR: Not found in file base/odls_base_default_fns.c at line 1076
b0618dfdd2d9
--------------------------------------------------------------------------
PRRTE has lost communication with a remote daemon.

  HNP daemon   : [[54984,0],0] on node b0618dfdd2d9
  Remote daemon: [[54984,0],1] on node d81fb6f346bb

This is usually due to either a failure of the TCP network
connection to the node, or possibly an internal failure of
the daemon itself. We cannot recover from this failure, and
therefore will terminate the job.
--------------------------------------------------------------------------
---------------------
Execution 10
---------------------
[b0618dfdd2d9:00656] [[54984,0],0] grpcomm:direct:send_relay proc [[54984,0],1] not running - cannot relay: NOT ALIVE 
[b0618dfdd2d9:00656] FORCE TERMINATE ORDERED AT grpcomm_direct.c:455 - error Unreachable(-12)
[b0618dfdd2d9:00656] [[54984,0],0] grpcomm:direct:send_relay proc [[54984,0],1] not running - cannot relay: NOT ALIVE 
[b0618dfdd2d9:00656] FORCE TERMINATE ORDERED AT grpcomm_direct.c:455 - error Unreachable(-12)
[b0618dfdd2d9:00656] [[54984,0],0] grpcomm:direct:send_relay proc [[54984,0],1] not running - cannot relay: NOT ALIVE 
[b0618dfdd2d9:00656] FORCE TERMINATE ORDERED AT grpcomm_direct.c:455 - error Unreachable(-12)
b0618dfdd2d9
[b0618dfdd2d9:00656] [[54984,0],0] grpcomm:direct:send_relay proc [[54984,0],1] not running - cannot relay: NOT ALIVE 
[b0618dfdd2d9:00656] FORCE TERMINATE ORDERED AT grpcomm_direct.c:455 - error Unreachable(-12)
[b0618dfdd2d9:00656] [[54984,0],0] grpcomm:direct:send_relay proc [[54984,0],1] not running - cannot relay: NOT ALIVE 
[b0618dfdd2d9:00656] FORCE TERMINATE ORDERED AT grpcomm_direct.c:455 - error Unreachable(-12)
ERROR: Incorrect number of lines of output. Expected 10. Actual 21
TERMINATING DVM...[b0618dfdd2d9:00656] [[54984,0],0] grpcomm:direct:send_relay proc [[54984,0],1] not running - cannot relay: NOT ALIVE 
[b0618dfdd2d9:00656] FORCE TERMINATE ORDERED AT grpcomm_direct.c:455 - error Unreachable(-12)
```